ksql-engine/src/main/java/io/confluent/ksql/function/UdfCompiler.java

```diff
@@ -167,6 +158,7 @@ private static String typeConversionCode(final String type, final int index) {
     builder.append(THROWS_TEMPLATE);
     return builder.toString()
         .replaceAll("#TYPE", type)
+        .replaceAll("#IS_PRIMITIVE", String.valueOf(isPrimitive))
```

Comment:
Is this needed any more? I guess it doesn't do any harm.

Reply:
probably not, but as you say no harm in keeping it.